test/new-e2e/tests/windows/install-test/install_test.go

```diff
@@ -0,0 +1,47 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+package installscript
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/DataDog/test-infra-definitions/scenarios/aws/vm/ec2os"
+	"github.com/DataDog/test-infra-definitions/scenarios/aws/vm/ec2params"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e/client"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e/params"
+	windows "github.com/DataDog/datadog-agent/test/new-e2e/tests/windows"
+	windowsAgent "github.com/DataDog/datadog-agent/test/new-e2e/tests/windows/agent"
+
+	"testing"
+)
+
+type agentMSISuite struct {
+	e2e.Suite[e2e.VMEnv]
+}
+
+func TestMSI(t *testing.T) {
+	e2e.Run(t,
+		&agentMSISuite{},
+		e2e.EC2VMStackDef(ec2params.WithOS(ec2os.WindowsOS)),
+		params.WithStackName(fmt.Sprintf("msi-test-%v", os.Getenv("CI_PIPELINE_ID"))),
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
On the CI a stack name is prefixed by the ci unique id, aka the pipeline id:
```suggestion
```

https://github.com/DataDog/datadog-agent/blob/pducolin/APL-2662-make-env-lazy/test/new-e2e/pkg/runner/ci_profile.go#L74

Reply:
good to know, thanks, I had just copied it from the [linux install script test ](https://github.com/DataDog/datadog-agent/blob/main/test/new-e2e/tests/agent-platform/install-script/install_script_test.go#L102)

Reply:
I see ! Indeed we can remove it from there too, will take care of that